api-reference/10 UI Components/dxValidationSummary/3 Methods/refreshValidationGroup().md

```diff
@@ -3,7 +3,252 @@ id: dxValidationSummary.refreshValidationGroup()
+                .dxValidator({
+                    validationRules: [{ type: 'required' }],
+                    validationGroup: "group"
+                })
+            );
+        }
+
+        renderGroup();
+
+        const summary = $("#summary").dxValidationSummary({
+            validationGroup: "group"
+        }).dxValidationSummary('instance');
+
+        $("#validate-group").dxButton({
+            text: "Validate",
+            validationGroup: "group",
+            onClick: function(e) {
+                e.validationGroup.validate();
+            }
+        });
+
+        $("#remove-group").dxButton({
+            text: "Remove group",
+            onClick: function() {
+                $("#container").empty();
+            }
+        });
+
+        $("#render-group").dxButton({
+            text: "Render group",
+            onClick: function() {
+                renderGroup();
+                summary.refreshValidationGroup(); // Refresh the subscription to ValidationGroup after rendering
+            }
+        });
+  
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-validation-group #group>
+        <dx-text-box *ngIf="visible">
+            <dx-validator>
+                <dxi-validation-rule type="required">
+                </dxi-validation-rule>
+            </dx-validator>
+        </dx-text-box>
+        <dx-text-box *ngIf="visible">
+            <dx-validator>
+                <dxi-validation-rule type="required">
+                </dxi-validation-rule>
+            </dx-validator>
+        </dx-text-box>
+        <dx-validation-summary #summary></dx-validation-summary>
+    </dx-validation-group>
+    <dx-button
+        text="Validate group"
+        (onClick)="validateGroup()"
+    >
+    </dx-button>
+    <dx-button
+        text="Remove group"
+        (onClick)="removeGroup()"
+    >
+    </dx-button>
+    <dx-button
+        text="Render group"
+        (onClick)="renderGroup()"
+    >
+    </dx-button>
+
+    <!-- tab: app.component.ts -->
+    import { Component, ViewChild } from '@angular/core';
+    import { DxValidationGroupComponent, DxValidationSummaryComponent } from 'devextreme-angular';
+
+    //...
+
+    export class AppComponent {
+        @ViewChild('group', {static: false}) validationGroup!: DxValidationGroupComponent
+        @ViewChild('summary', {static: false}) summary!: DxValidationSummaryComponent
+        visible: boolean = true;
+        removeGroup() {
+            this.visible = false;
+        }
+        renderGroup() {
+            this.visible = true;
+            this.summary.instance.refreshValidationGroup(); // Refresh the subscription to ValidationGroup after rendering
+        }
+        validateGroup() {
+            this.validationGroup.instance.validate();
+        }
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div id="container">
+            <DxValidationGroup
+                :ref="groupRefKey"
```

Comment:
Do we really need to remember group ref to validation it? Maybe we can just call `e.validationGroup.validate()` inside of click handler?